scylla/benches/benchmark.rs

```diff
@@ -1,11 +1,11 @@
 use criterion::{criterion_group, criterion_main, Criterion};
 
 use bytes::BytesMut;
-use scylla::{
-    frame::types,
-    transport::partitioner::{calculate_token_for_partition_key, Murmur3Partitioner},
+use scylla::transport::partitioner::{calculate_token_for_partition_key, Murmur3Partitioner};
+use scylla_cql::{
+    frame::{response::result::ColumnType, types},
+    types::serialize::row::SerializedValues,
 };
-use scylla_cql::{frame::response::result::ColumnType, types::serialize::row::SerializedValues};
```

Comment:
Why is this change necessary? Apart from formatting / grouping, I see only 1 change - importing `scylla_cql::frame::types` instead of `scylla::frame::types`. but I see in `lib.rs` that `scylla::frame::types` still exists, so why change this file at all?

Reply:
@muzarski

Reply:
`scylla_cql::frame::types` has a lot of utility functions such as `read/write_int`. These should not be public. OTOH, it contains the definitions of `Consistency` and `SerialConsistency` - these obviously should be public. This is the change we introduce to `scylla/src/lib.rs`.

Why do we need the change in `scylla/benches/benchmark.rs`? The answer is that we cannot use private definitions of `scylla` crate there. Benchmarks use some of the utility functions that I just mentioned above (which were just made private). This is why we change `scylla::frame::types` to `scylla_cql::frame::types` (in this crate, these utility functions are public).